src/lib/misc.ts

```diff
@@ -201,8 +201,9 @@ export function getOutcomeTranslation(outcome: string) {
     }
 
     if (/[0-9.]+/.test(outcome)) {
-        const num = outcome.match(/([0-9.]+)/)[1];
-        return interpolate(pgettext("Game outcome", "{{number}} points"), { number: num }); // eslint-disable-line id-denylist
+        const num: number = +outcome.match(/([0-9.]+)/)[1];
+        const rounded_num = Math.round(num * 2) / 2;
+        return interpolate(pgettext("Game outcome", "{{score}} points"), { score: rounded_num });
```

Comment:
Let's change this back to "{{number}} points", since that string is already translated - replacing it with "{{score}} points" will cause it to show up in english until translators come along and re-translate it.

Reply:
Hah that was my fault for suggesting the change 😊